lnrpc/rpc.proto

```diff
@@ -2909,12 +2909,39 @@ message ListPaymentsRequest {
     that pending payments, as well as failed payments will show up if this
     field is set to True.
     */
-    bool include_incomplete = 1;
+    bool include_incomplete = 1 [json_name = "include_incomplete"];
+
+    /**
+    The index of a payment that will be used as either the start or end of a
+    query to determine which payments should be returned in the response.
+    */
```

Comment:
Document that it is zero-based?

Reply:
Probably also mention how the index works together with `include_incomplete`. Does this filter change the indexing?

Reply:
Actually it's one-based, only zero has a special meaning. The filter doesn't change the index meaning, documented that in the rpc as well as in lncli.